fetcher/server.go

```diff
@@ -9,8 +9,8 @@ import (
 var _ Fetcher = &server{}
 
 // NewServer creates a new fetcher that will collect pricing information on servers.
-func NewServer(pricing *PriceProvider) Fetcher {
-	return &server{newBase(pricing, "server", "location", "type")}
+func NewServer(pricing *PriceProvider, additionalLabels ...string) Fetcher {
+	return &loadBalancer{newBase(pricing, "server", []string{"location", "type"}, additionalLabels...)}
```

Comment:
I totally missed this one, but thankfully the test caught it 😄. This copy-paste error happend to me a few times at well... I will fix it, so no worries.